src/cpg_flow_gatk_sv/config_template.toml

```diff
@@ -151,10 +151,17 @@ scramble_docker = "australia-southeast1-docker.pkg.dev/cpg-common/images/sv/scra
 str = "australia-southeast1-docker.pkg.dev/cpg-common/images/sv/str:2023-05-23-v0.27.3-beta-e537bdd6"
 sv_base_docker = "australia-southeast1-docker.pkg.dev/cpg-common/images/sv/sv-base:2024-10-25-v0.29-beta-5ea22a52"
 sv_base_mini_docker = "australia-southeast1-docker.pkg.dev/cpg-common/images/sv/sv-base-mini:2024-10-25-v0.29-beta-5ea22a52"
-sv_pipeline_docker = "australia-southeast1-docker.pkg.dev/cpg-common/images/sv/sv-pipeline:2025-03-25-v1.0.2-hotfix-65eddd78"
-sv_pipeline_qc_docker = "australia-southeast1-docker.pkg.dev/cpg-common/images/sv/sv-pipeline:2025-03-25-v1.0.2-hotfix-65eddd78"
+sv_pipeline_docker = "australia-southeast1-docker.pkg.dev/cpg-common/images/sv/sv-pipeline:2025-06-27-v1.0.4-63e6c81e"
+sv_pipeline_qc_docker = "australia-southeast1-docker.pkg.dev/cpg-common/images/sv/sv-pipeline:2025-06-27-v1.0.4-63e6c81e"
 wham_docker = "australia-southeast1-docker.pkg.dev/cpg-common/images/sv/wham:2024-10-25-v0.29-beta-5ea22a52"
 strvctvre = "australia-southeast1-docker.pkg.dev/cpg-common/images/strvctvre:v.1.10"
+vapor_docker = "australia-southeast1-docker.pkg.dev/cpg-common/images/sv/vapor:header-hash-2fc8f12"
+sv-base-virtual-env = "australia-southeast1-docker.pkg.dev/cpg-common/images/sv/sv-base-virtual-env:5994670"
+cnmops-virtual-env = "australia-southeast1-docker.pkg.dev/cpg-common/images/sv/cnmops-virtual-env:5994670"
+sv-pipeline-virtual-env = "australia-southeast1-docker.pkg.dev/cpg-common/images/sv/sv-pipeline-virtual-env:2025-05-01-v1.0.3-85054e07"
+samtools-cloud-virtual-env = "australia-southeast1-docker.pkg.dev/cpg-common/images/sv/samtools-cloud-virtual-env:5994670"
+sv-utils-env = "australia-southeast1-docker.pkg.dev/cpg-common/images/sv/sv-utils-env:2023-02-01-v0.26.8-beta-9b25c72d"
```

Comment:
As an FYI I don't think these images are used, except for in the docker building workflow as intermediates, I should have removed them from the copy script...